sdk/iot/Azure.Iot.Hub.Service/samples/IotHubClientSamples/BulkDeviceIdentityLifecycleSamples.cs

```diff
@@ -98,6 +98,8 @@ public async Task CreateDeviceIdentitiesAsync(IEnumerable<DeviceIdentity> device
             }
             catch (Exception ex)
             {
+                // Try to cleanup before exiting with fatal error.
+                await CleanupHelper.DeleteAllDevicesInHubAsync(IoTHubServiceClient);
```

Comment:
This is to ensure we cleanup all devices created in the hub even if there was a fatal error.